packages/core/src/services/ai/helpers.ts

```diff
@@ -155,7 +168,6 @@ export function createProvider({
       return Result.ok(
         createGoogleGenerativeAI({
           apiKey,
-          ...(config?.google ?? {}),
```

Comment:
The data here was not related to this provider. I think it was from the past. Let me know if I'm wrong 🙏